### DISCUSSION TITLE
macOS Monterey 12.5.1 - "Rancher Desktop" is damaged and can't be opened.

### DISCUSSION THREAD

**Discussion by carolgschwend** [rancher-sandbox/rancher-desktop#2792] [Q&A] [answered]
After upgrading to macOS Monterey 12.5.1 today I got a fatal error upon starting rancher.  After uninstalling rancher, I'm not able to open and install new downloaded rancher files.  Can anyone verify rancher works with Monterey 12.5.1 ?

Fatal Error
![image](https://user-images.githubusercontent.com/21066111/185808511-9d7d170e-526a-454c-9192-ef1570789be6.png)

Damaged and can't be opened
![image](https://user-images.githubusercontent.com/21066111/185808532-80106c3c-be57-4cbe-92ca-96dd3d4a27ce.png)

**Comment by boogi** [comment]
Same here. Though I first updated to MacOS 12.5.1 few days ago, and only yesterday to Rancher 1.5.1. One interesting thing I see is that I've got M1 (aarch, arm64 arch), but after downloading limactl manually and putting it in /Applications/Rancher/.../lima/bin/limactl I get error as in printscreen, so maybe in this build just architectures were messed up? (limactl probably needed only on M1 to run x86 containers, and not needed on x86 Intel cpu's so that's why it's not there - just a guess).
<img width="768" alt="Screenshot 2022-08-23 at 14 22 07" src="https://user-images.githubusercontent.com/6509310/186158859-f0457645-c398-441e-bcfd-57203b9ec0e9.png">

EDIT: Also tried installing 1.5.1 via brew, then downloading .dmg for 1.5.1 and 1.5.0 and mounting (limactl is there) and installing (got "Rancher Desktop is damaged" error as OP, then 1.5.1 and 1.5.0 tar.gz files unpacking (same error), finally settled on downloading 1.4.1 that worked previously - only to get the same error :o

**Reply by boogi** [reply]
Not moving unzipped Rancher Desktop to /Application, and running it from e.g. ~/Downloads, allows it to run properly.
As it is discussed in https://github.com/rancher-sandbox/rancher-desktop/issues/2803 it's just one of MacOS 12.5.1 quirks.
EDIT: seems like Apple fixed it, and Rancher Desktop (or rather part of it, limactl) is not seen as a virus anymore, so moving it to /Application works for me right now.

**Comment by jandubois** [comment, marked as answer]
This seems to happen due to broken ClamAV virus definitions; updating them should fix your problem: https://github.com/lima-vm/lima/issues/1028
